### PR TITLE
docs: corrected grammatical errors in README.md

### PR DESCRIPTION
**Pull request by alienishi**
I have corrected all the grammatical errors in README.md.

### CODE REVIEW

**Comment by alienishi** [comment]
@ajith-k-v
Please review my PR, and if according to the guidelines please do merge it :)

**Comment by alienishi** [comment]
@ajith-k-v
@karan-rathod-316 
Please review my PR, and if according to the guidelines please do merge it :)

File: README.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ ToolJet is an **open-source low-code framework** to build and deploy internal to
...
+You can deploy ToolJet on Heroku using one-click deployment.
 
 <p align="center">
   <a href="https://heroku.com/deploy?template=https://github.com/tooljet/tooljet/tree/main"><img src="https://www.herokucdn.com/deploy/button.svg" alt="Deploy to Heroku" height=32></a>
   <a href="https://cloud.digitalocean.com/apps/new?repo=https://github.com/ToolJet/ToolJet/tree/main"><img src="https://www.deploytodo.com/do-btn-blue.svg" alt="Deploy to DigitalOcean" height=32></a>
 </p>
 
 ### Try using Docker
-Want to give ToolJet a quick spin on your local machine? You can run the following command from your terminal to have ToolJet up and running right away.
+Want to give ToolJet a quick spin on your local machine? You can immediately run the following command from your terminal to have ToolJet up.
```
Comment (karan-rathod-316) [inline README.md:81]: This change is not needed.

File: README.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ You can use ToolJet Cloud for a fully managed solution. If you want to self-host
 | Deploying ToolJet on a Subpath  | [Link](https://docs.tooljet.com/docs/setup/tooljet-subpath/)   |
 
 ## Marketplace 
-ToolJet can now be found on both AWS and Azure Marketplaces, making it simpler than ever to access and deploy our app-building platform.
+ToolJet can now be found on both AWS and Azure Marketplaces, making accessing and deploying our app-building platform simpler than ever.
```
Comment (karan-rathod-316) [inline README.md:126]: This change is not needed.

File: README.md [hunk 3]
```diff
@@ -132,10 +132,10 @@ For general help using ToolJet, please refer to the official [documentation](htt
 
 - [Slack](https://tooljet.com/slack) - Discussions with the community and the team.
 - [GitHub](https://github.com/ToolJet/ToolJet/issues) - For bug reports and feature requests.
-- [Twitter](https://twitter.com/ToolJet) - Get the product updates easily.
+- [Twitter](https://twitter.com/ToolJet) - Get the product updates quickly.
 
 ## Roadmap
-Check out our [roadmap](https://github.com/ToolJet/ToolJet/projects/2) to stay updated on recently released features and to learn about what's coming next.
+Check out our [roadmap](https://github.com/ToolJet/ToolJet/projects/2) to stay updated on recently released features and learn about what's coming.
```
Comment (karan-rathod-316) [inline README.md:138]: Rephrase this to :

Check out our [roadmap](https://github.com/ToolJet/ToolJet/projects/2) to stay updated on recently released features and learn about what's coming next.

**Review by karan-rathod-316 — COMMENTED** [review]
Please make the changes based on the comments.

**Comment by tejakummarikuntla** [comment]
Hey @alienishi, could you please address the reviews to progress the PR?

**Comment by alienishi** [comment]
@karan-rathod-316 Thanks for the guidance will do the changes right away :)

**Comment by alienishi** [comment]
@karan-rathod-316 
I have made the suggested changes 
Kindly verify it and if according to the guidelines please do merge it :)

**Review by karan-rathod-316 — APPROVED** [review]
LGTM

**Comment by alienishi** [comment]
@karan-rathod-316 
Thanks!
Please do merge it :)
